app/views/employers/commitments.html.erb

```diff
@@ -1,32 +1,51 @@
 <% provide(:title, "Commitments") %>
 <div id="content" class="interior">
   <a href="#content" class="sr-only">Content starts here.</a>
-
-  <div class="splash">
-    <div class="row">
-      <div class="small-12 columns">
-        <ul class="breadcrumbs" role="menubar" aria-label="Primary">
-          <li class="parent"><%= link_to "Careers and Employment", root_path %></li>
-          <li class="parent"><%= link_to "Employers", employer_home_path %></li>
-	  <li class="active">Commitments</li>
-        </ul>
-      </div>
-    </div>
-  </div>
-
+  <nav class="va-nav-breadcrumbs">
+    <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
+      <li><a href="/">Home</a></li>
+      <li><%= link_to "Careers and Employment", root_path %></li>
+      <li class="parent"><%= link_to "Employers", employer_home_path %></li>
+      <li class="active">Commitments</li>
+    </ul>
+  </nav>
   <div class="section one">
     <% unless user_signed_in? && current_user.is_employer? %>
       <div class="row">
-        <div class="small-12 medium-8 columns">
-          <div class="primary">
-            <p>Veterans and military family members have consistently proven themselves to be reliable, independent, and hardworking team members. Veterans also have soft skills such as problem solving and creativity that can benefit a company’s bottom line. By signing up and making a commitment, your company will stand out to candidates eager to enter the civilian workforce.</p>
-          </div>
+        <div class="small-12 medium-7 columns">
+          <h1>Commitments</h1>
+          <p>Veterans and military family members have consistently proven themselves to be reliable, independent, and hardworking team members. Veterans also have soft skills such as problem solving and creativity that can benefit a company’s bottom line. By signing up and making a commitment, your company will stand out to candidates eager to enter the civilian workforce.</p>
+        </div>
+        <% if user_signed_in? && current_user.is_employer? %>
```

Comment:
This line an most of the below used to be outside the `unless` statement above, but is now inside. This logic error is what's making the tests fail.